src/IssueViz.Security.UnitTests/OpenInIDE/Api/OpenInIDERequestHandlerTests.cs

```diff
@@ -314,6 +321,14 @@ private static IAnalysisIssueVisualization CreateHotspotVisualization(string fil
         private void SetSelectionServiceExpectedItem(IAnalysisIssueVisualization expected) =>
             selectionServiceMock.Setup(x => x.Select(expected));
 
+        private void CheckInvariants()
```

Comment:
Maybe VerifyPersistentBehavior ? CheckConstantBehavior? 
I think Invariants give me a "variables" feeling, as in, you're asserting numbers or strings (probably due to `StringComparer.InvariantCulture`...)

Reply:
To me, "Persistent" suggests serialization and "Constant" suggests data. I've changed it to "CheckInvariantBehaviours".